src/bindgen/ir/ty.rs

```diff
@@ -687,15 +688,54 @@ impl Type {
         generic_params: &GenericParams,
         library: &Library,
         out: &mut Dependencies,
+        ptr_types: &mut IndexSet<GenericPath>,
+
```

Comment:
nit: space. But also rather than having an `is_ptr` param, it seems this should be handled in `Type::Ptr` itself?